Kickstarter-iOS/Features/ProjectPage/Views/Cells/ProjectTabDisclaimerCell.swift

```diff
@@ -116,40 +116,53 @@ final class ProjectTabDisclaimerCell: UITableViewCell, ValueCell {
       And that way could just do: `Strings.To_learn_how_Kickstarter_encourages_sustainable_practices(environmental_resources_link: environmentLink)` after line 120.
      **/
 
-    switch type {
-    case .aiDisclosure,
-         .environmental:
-      let learnMoreString = NSMutableAttributedString(
+    let additionalText = type == .environmental ?
+      NSAttributedString(
         string: Strings.To_learn_how_Kickstarter_encourages_sustainable_practices(),
         attributes: regularFontAttribute
-      )
+      ) :
+      NSAttributedString(string: "")
 
+    var linkString: String
+    let baseUrl = AppEnvironment.current.apiService.serverConfig.webBaseUrl
+    switch type {
+    case .aiDisclosure:
+      guard let link = HelpType.aiDisclosure
+        .url(withBaseUrl: baseUrl)?.absoluteString else {
+        return additionalText
+      }
+      // FIXME: Use translatable string.
+      linkString = "<a href=\"\(link)\">Learn about AI policy on Kickstarter</a>"
+    case .environmental:
       guard let environmentLink = HelpType.environment
-        .url(withBaseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl)?.absoluteString else {
-        return learnMoreString
+        .url(withBaseUrl: baseUrl)?.absoluteString else {
+        return additionalText
```

Comment:
Instead of potentially returning an empty String here, would it be nicer to leave the `case .environmental` be and update only the `aiDisclosure` case.

Afaik, the changes at this point would be simple:
```
let defaultAIPolicyText = NSMutableAttributedString(
        string: "Learn about AI policy on Kickstarter",
        attributes: regularFontAttribute
      )

 guard let aiLink = HelpType.aiDisclosure
        .url(withBaseUrl: AppEnvironment.current.apiService.serverConfig.webBaseUrl)?.absoluteString else {
        return defaultAIPolicyText
      }

let aiDisclosureString = "<a href=\"\(aiLink)\">Learn about AI policy on Kickstarter</a>"

      guard let aiDisclosureAttributedString = try? NSMutableAttributedString(
        data: Data(aiDisclosureString.utf8),
        options: [
          .documentType: NSAttributedString.DocumentType.html,
          .characterEncoding: String.Encoding.utf8.rawValue
        ],
        documentAttributes: nil
      ) else { return defaultAIPolicyText }

      let fullRange = (aiDisclosureAttributedString.string as NSString)
        .range(of: aiDisclosureAttributedString.string)
      aiDisclosureAttributedString.addAttributes(coloredFontAttribute, range: fullRange)

return aiDisclosureString
```

Seems a bit repetitive because of the guard, but maybe we can extract just that out instead?